src/main/java/joeduck/utils/Utils.java

```diff
@@ -0,0 +1,36 @@
+package joeduck.utils;
+
+import joeduck.task.Task;
+
+import java.util.List;
+
+/**
+ * Helper functions not specific to any other class.
+ */
+public class Utils {
+
+    /**
+     * Gets the string representation of a List of Task.
+     * If forPrinting is true, will get the equivalent representation meant for writing to tasks.txt.
+     * Else, will get the representation meant for printing to the user.
+     * @param list List of Task to be converted to String.
+     * @param forPrinting Boolean representing if the String is for showing to the user.
+     * @return String representation of the List of Task.
+     */
+    public static String inputsToString(List<Task> list, boolean forPrinting) {
```

Comment:
Perhaps this method could be changed to better reflect what it does?
```suggestion
    public static String getTasksAsString(List<Task> tasks, boolean forPrinting) {
```
Minor nit: would this be better as a instance method of `TaskList`?